src/client/containers/CategoryPage/CategoryPage.Style.css

```diff
@@ -1,3 +1,16 @@
 h1 {
-  color: aqua;
+  color: rgb(0, 0, 0);
+  text-align: left;
+  font-size: 3em;
+  margin-left: 180px;
+  margin-top: 60px;
+  text-transform: uppercase;
+}
+
+.sorting-div {
+  display: none;
+}
+
+h3 {
+  visibility: hidden;
```

Comment:
Please don't hide stuff or generally apply CSS to other components than the current one. It will make it difficult to track why that component suddenly changes behaviour. Instead, add props that can configure the other component or talk to whoever built that component and discuss how it can be configure to work in your setting.

Reply:
done